src/components/ItemCard.js

```diff
@@ -0,0 +1,12 @@
+import "./ItemCard.css";
+
+export default function ItemCard() {
+  return (
+    <div className="Item-card frame">
+      <li className="Item-card">
+        <img src="image" alt="description" />
+        <h3 className="Item-card__name">Chocolate cake</h3>
```

Comment:
I did not see an h2 on your page so this could be the h2 then. :)))

Reply:
I agree, you should not leave out headings.

Reply:
Thought ```h2``` would be too big to appear on the card. But now as I check it again ```h2``` suits it better.